### ISSUE TITLE
LifecycleOwner is attempting to register while current state is RESUMED. LifecycleOwners must call register before they are STARTED.

### ISSUE THREAD

**Issue by WenLonG12345** [AnilFurkanOkcun/UWMediaPicker-Android#14]
Hi, I faced a problem recently when I update my build.gradle (app) with the new Activity Result API. 

```
implementation "androidx.activity:activity-ktx:1.1.0"
implementation 'androidx.fragment:fragment-ktx:1.3.0-beta02'
```

however, after I implement these 2 dependencies, I experience a crashing when open gallery with this fatal error. 
```
LifecycleOwner is attempting to register while current state is RESUMED. LifecycleOwners must call register before they are STARTED.
```

I double check the sample project build.gradle(app). It seems there is no any implementation of ```androidx.acitivty``` and ```androidx.fragment```. 

Please give me some guide on how to fix this. Thanks in advance :) 
PS: this is a really nicely design library 👍

**Comment by AnilFurkanOkcun** [comment]
Hello, Thank you for reaching out. After I used Activity Result API, they changed things. So it's my fault to use beta dependency in a library. Sorry about that, I was very excited about Activity Result API. 

I fixed the issue in 1.3.2 version. You can try it :)
`implementation 'com.github.AnilFurkanOkcun:UWMediaPicker-Android:1.3.2'`

**Comment by WenLonG12345** [comment]
Thanks for the fix! 
It is ok since things in Android changed and deprecated easily. Nice library to use anyway :)
but it is sad to say goodbye to this library when I migrating to sdk 30, no more 3rd party file picker can be used already :(
